core/src/main/java/io/questdb/griffin/model/IntervalUtils.java

```diff
@@ -534,6 +536,22 @@ public static void parseIntervalEx(CharSequence seq, int lo, int lim, int positi
             }
         }
```

Comment:
Please reformat code using qdb setttings . 
IntelliJ should pick up the config automatically but if you don't have reformat & rearrange turn on in 'Actions on save' then you've to select IntervalUtils and choose Code | Reformat File on .